test/cppsuite/tests/bounded_cursor_stress.cpp

```diff
@@ -225,16 +224,19 @@ class cursor_bound_01 : public test {
                     upper_bound = bound(tc->key_size, false, lower_bound.get_key()[0] + 1);
```

Comment:
So i just realised that we can place a ternary `? :` operator here for reverse collator, could you try that out here?